<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="paidOutThisWeekLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Потрачено на этой неделе:</string>
    </property>
   </widget>
   <widget class="QLabel" name="remainedThisWeekLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Осталось на этой неделе:</string>
    </property>
   </widget>
   <widget class="QLabel" name="paidOutThisWeekNum">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="remainedThisWeekNum">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>701</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="addExpensesTextEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="addExpensesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить траты из текста -&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addExpensesTextBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addExpensesFileBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить траты из файла</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="FoodLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Еда</string>
    </property>
   </widget>
   <widget class="QLabel" name="ExtraLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Внеучетные</string>
    </property>
   </widget>
   <widget class="QToolButton" name="rightToolButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::ArrowType::RightArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="leftToolButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::ArrowType::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QScrollArea" name="ScrollAreaExpenses">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>360</width>
      <height>191</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="gridLayoutExpenses">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>358</width>
       <height>189</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="removeExpensesBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>530</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addExpensesToDBBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>530</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Добавить трату</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QScrollArea" name="ScrollAreaExtraExpenses">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>360</width>
      <height>191</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="gridLayoutExtraExpenses">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>358</width>
       <height>189</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addExpensesToColumns">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Внести траты в базу</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="EUR_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GEL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="USD_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>35</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>USD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RUB_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>RUB:</string>
    </property>
   </widget>
   <widget class="QLabel" name="GEL_value_label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="USD_value_label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>35</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RUB_value_label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>50</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>3</width>
      <height>50</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="unfixExpensesThisWeekNum">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="unfixExpensesThisWeekLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Внеучетные траты на неделе:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lizaExpensesThisWeekLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Траты Лизочки на неделе:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lizaExpensesThisWeekNum">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ExpensesThisMonthLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Всего за месяц:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ExpensesThisMonthNum">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>181</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="monthDatesLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>180</width>
      <height>11</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>627</x>
      <y>250</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ExpensesThisMonthLabel_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Всего:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ExpensesThisMonthNumOverall">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="monthlyRentLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Арендная плата:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="monthlyRentNum">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>600</string>
    </property>
   </widget>
   <widget class="QComboBox" name="monthlyRentCurrencyComboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>USD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="remainDebtNum">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-3704.13</string>
    </property>
   </widget>
   <widget class="QLabel" name="remainDebtLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Оставшийся долг:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcDebtBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="yearComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="monthComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>January</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>February</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>April</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>August</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>September</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>October</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>November</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>December</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="weekComboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
